docs/quality-gates/IntegrationWithCICD/IntegrationWithCircleCI.md

```diff
@@ -0,0 +1,88 @@
+---
+sidebar_label: Integration with CircleCI
+---
+
+# Integration with CircleCI
+
+This guide will walk you through the process of integrating ReportPortal with CircleCI for Node.js projects. We'll define a job in the `.circleci/config.yml` file. The job includes steps for application build, running tests, capturing ReportPortal Launch ID, and implementing a Quality Gate check.
+
+## Prerequisites
+
+Before you start, make sure you have:
+
+- A CircleCI account and a project set up with a `.circleci/config.yml` file.
+- A ReportPortal account and a project created.
+
+## CircleCI Job Overview
+
+A CircleCI job for integrating ReportPortal includes the following key steps:
+
+1. **Run application build and tests:** Execute npm tests.
+2. **Capture Launch ID:** Extract the ReportPortal Launch ID from the output logs of the test execution.
+3. **Quality Gate:** Check the Quality Gate status on ReportPortal using the Launch ID captured in Step 2.
+
+## Configuring CircleCI pipeline
+
+### Environment Variables
+
+Define these environment variables in your CircleCI project settings:
+
+- `RP_INSTANCE_URL`: The URL of your ReportPortal instance.
+- `RP_PROJECT`: The name of your project as it is in ReportPortal.
+- `RP_API_KEY`: Your API key from ReportPortal.
+- `REQUEST_TIMEOUT_SECONDS`: The timeout duration for ReportPortal API requests.
+- `SCRIPT_TIMEOUT_SECONDS`: The timeout duration for the entire script execution.
+
+### CircleCI Configuration
```

Comment:
Please split it into steps with explanations as in other articles.

Reply:
Fixed please check